test/image/mocks/colorscale_opacity.json

```diff
@@ -0,0 +1,56 @@
+{
+    "data": [
+        {
+            "y": [
+                5,
+                5,
+                5,
+                5
+            ],
+            "mode": "markers",
+            "marker": {
+                "size": 40,
+                "colorscale": [
+                    [
+                        0,
+                        "rgb(255,0.0,0.0)"
+                    ],
+                    [
+                        1,
+                        "rgba(0.0,0.0,255,0.5)"
+                    ]
+                ],
+                "color": [
+                    0,
+                    1,
+                    2,
+                    3
+                ],
+                "cmin": 0,
+                "cmax": 3
```

Comment:
I'm curious to see how the colorbar looks in this case.

Could you add a color bar to this mock? Adding `marker.color.colorbar = {};` should do the trick.